nservicebus/azure/azure-servicebus-transport.md

```diff
@@ -12,33 +12,33 @@ redirects:
+Azure Service Bus (ASB) is a messaging infrastructure that sits between applications, allowing them to exchange messages in a loosely coupled fashion for improved scale and resiliency. Service Bus Queues offer simple first in, first out guaranteed message delivery and support a range of standard protocols (REST, AMQP, WS*) and APIs (to put/pull messages on/off the queue). Service Bus Topics deliver messages to multiple subscribers and easily fan out message delivery at scale to downstream systems.
 
- * The main advantage of this service is that it offers a highly reliable and (relatively) low latency remote messaging infrastructure. A single queue message can be up to 256 KB in size, and a queue can contain many messages, up to 5GB in total. Furthermore it is capable of emulating local transactions using its queue peek-lock mechanism and has many built-in features that you (and NServiceBus) can take advantage of, such as message deduplication and deferred messages.
- * The main disadvantage of this service is its dependency on TCP (if you want low latency), which may require you to open some outbound ports on your firewall. Additionally, the price may be steep, depending on your scenario ($1 per million messages).
-
-Note: Publish/Subscribe and Timeouts (including message deferral) are supported natively by the transport and do not use persistence.
+ * The main advantage of ASB is that it offers a highly reliable and (relatively) low latency remote messaging infrastructure. A single message can be up to 256 KB in size, and a queue can keep many messages at once, up to 5 GB size in total. Furthermore it is capable of emulating local transactions using its queue peek-lock mechanism and has many built-in features that users (and NServiceBus) can take advantage of, such as message deduplication and deferred messages.
+ * The main disadvantage of ASB is its dependency on TCP (for low latency), which may require opening some outbound ports on the firewall. Additionally, in some systems the price may get high ($1 per million messages).
 
+Note: Publish/Subscribe and Timeouts (including message deferral) are supported natively by the ASB transport and do not use persistence.
 
 ## Enabling the Transport
 
-First, ensure that Standard Messaging Tier is being used for Azure Service Bus when creating the namespace in the Azure portal.
+Firstly, choose Standard Messaging Tier for Azure Service Bus when creating the namespace at Azure portal.
 
-Second, reference the assembly that contains the Azure Service Bus transport definition. The recommended method is to add a NuGet package reference to the `NServiceBus.Azure.Transports.WindowsAzureServiceBus` package to the project.
+Secondly, reference the assembly that contains the Azure Service Bus transport definition. The recommended method is to add a `NServiceBus.Azure.Transports.WindowsAzureServiceBus` NuGet package reference to the project.
```

Comment:
> reference the assembly that contains the Azure Service 

Should it be "reference `NServiceBus.Azure.Transports.WindowsAzureServiceBus` nuget package only"?